docs/docs/dev_docs/tutorials/writing_private_voting_contract.md

```diff
@@ -0,0 +1,251 @@
+import Image from '@theme/IdealImage';
+
+# Writing a private voting smart contract in Aztec.nr
+
+In this tutorial we will go through writing a very simple private voting smart contract in Aztec.nr. You will learn about private functions, public functions, composability between them, and state management.
+
+We will build this:
+
+<Image img={require('/img/tutorials/voting_flow.png')} />
+
+* The contract will be initiated with an admin, stored publicly
+* A voter can vote privately, which will call a public function and update the votes publically. This keeps who they are, what they have voted for, and whether or not they have voted private - but the vote itself public
```

Comment:
>  This keeps who they are, what they have voted for

Unsure what you are trying to stay and looks like we are leaking privacy when your contract isn't. You simply store the tally of votes per candidate. The action of voting is public but who initiated the vote isn't

Reply:
haha yes trying to say it keeps that stuff private. but good call will change